sftpgo/repository.yaml

```diff
@@ -0,0 +1,10 @@
+name: sftpgo
+description: Fully featured and highly configurable SFTP server with optional HTTP/S, FTP/S and WebDAV support - S3, Google Cloud Storage, Azure Blob
+category: DATA
+private: true
```

Comment:
we'll need this to be `private: false` if you want to be able to iterate on it quickly